crates/esthri-cli/src/http_server.rs

```diff
@@ -224,10 +226,12 @@ fn with_s3_client(
 pub fn esthri_filter(
```

Comment:
Please add a comment describing this function and this filters use cases

Reply:
done